tusclient/uploader.py

```diff
@@ -61,6 +63,17 @@ class Uploader(object):
         - retry_delay (int):
             How long (in seconds) the uploader should wait before retrying a failed upload attempt.
             If not specified, it defaults to 30.
+        - resumable (bool):
```

Comment:
I am aware that the tus-js-client's option is named the same, but I would suggest renaming it to `storeURLs` or similar which is more expressive. I would love to do this for the tus-js-client but it's a breaking chance and is therefore not worth it.

Reply:
agreed 👍